### PR TITLE
run tests in github actions

### PR DESCRIPTION
**Pull request by kd7lxl**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          # the test fails with these present
+          rm -fv \
+            tests/images/AnyTone_778UV.img \
+            tests/images/CRT_Micron_UV.img \
+            tests/images/Kenwood_TK-3180K2.img \
+            tests/images/Kenwood_TK-8180.img \
+            tests/images/Kenwood_TM-D710G_CloneMode.img \
+            tests/images/Kenwood_TM-D710_CloneMode.img \
+            tests/images/Midland_DBR2500.img \
+            tests/images/Retevis_RT95.img
```
Comment (kd7lxl) [inline .github/workflows/test.yaml:31]: This is some of my best work.

**Comment by kk7ds** [comment]
This is cool, but if you could just install the future module, you should be able to get those without having to nuke the images. Obviously want to try to avoid that if possible.

Also, ideal to run with tox if possible because it constructs a specific environment and makes it easier to reproduce locally for people. That commit I linked to had a couple of tweaks to make that work.

**Comment by kk7ds** [comment]
If you rebase this on master, I think it'll pass now.

**Comment by kd7lxl** [comment]
> If you rebase this on master, I think it'll pass now.

The test result is here:
https://github.com/kd7lxl/chirp/runs/3395571267?check_suite_focus=true

Since it's a fork, I don't think it'll report build status here until it's approved. After it's merged, fork PRs will show up like this:
https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks

**Comment by kk7ds** [comment]
```unit: commands succeeded
  driver: commands succeeded
  style: commands succeeded
  congratulations :)
```

**Pull request merged**